packages/node_modules/@node-red/editor-client/src/js/ui/view.js

```diff
@@ -1823,6 +1831,29 @@ RED.view = (function() {
         return [offsetW+w,offsetH+h];
     }
 
+    function convertLineBreakCharacter(str) {
```

Comment:
I can't work out what this function is doing.

Why is `split('\\n')` not enough to split the text into multiple lines? Can you provide some examples that require this extra processing?

Reply:
I implemented to show '\n' when you type '\\\n'. For example, when you type a path such as 'c:\node-red\node-red',  automatically the path is divided in 3 lines. If you want to avoid this, type backslash escape sequences like this 'c:\\\node-red\\\node-red'.